internal/website/blog/2022-07-06-the-future-of-faust.md

```diff
@@ -0,0 +1,38 @@
+---
+title: The Future of Faust.js
+description: An update on Faust.js, where it is and where it is going
+slug: the-future-of-faust
+hide_table_of_contents: true
+---
+
+We launched this blog 7 months ago with a goal of keeping you informed of what is going on in the development of Faust.js as well as to help you get started using the framework in your own projects. To date, we simply have not prioritized keeping up here on the blog and it is time for that to change.
+
+<!--truncate-->
+
+Today's we want to talk about where Faust.js is today and where we're working on taking it in the future. It will be the first of many regular posts designed to tell the story of Faust.js and how it affects developers of all backgrounds.
```

Comment:
```suggestion
Today we want to talk about where Faust.js is and where we're working on taking it in the future. It will be the first of many regular posts designed to tell the story of Faust.js and how it affects developers of all backgrounds.
```